cli/util/uploader.go

```diff
@@ -397,9 +415,11 @@ func updateIndexFiles(ctx *cli.Context, p *pool.Pool, containerID cid.ID, accoun
 			*object.NewAttribute(attributeKey, strconv.Itoa(int(i))),
 			*object.NewAttribute("size", strconv.Itoa(int(indexFileSize))),
 		}
-		err = uploadObj(ctx.Context, p, signer, account.PrivateKey().GetScriptHash(), containerID, buffer, attrs, homomorphicHashingDisabled)
-		if err != nil {
-			return fmt.Errorf("failed to upload index file %d: %w", i, err)
+		errRetr := retry(func() error {
+			return uploadObj(ctx.Context, p, signer, account.PrivateKey().GetScriptHash(), containerID, buffer, attrs, homomorphicHashingDisabled)
+		})
+		if errRetr != nil {
+			return fmt.Errorf("failed to upload index file %d: %w", i, errRetr)
```

Comment:
Useless change, revert it.

Reply:
Why? In case of failure of uploading it won't retry. In case of block uploading we do have retry on it. I can put it in a separate commit.

Reply:
Retry itself is OK, I mean `err` -> `errRtr` change, there's no bug in this code related to `err`.

Reply:
Regarding retry: yes, put it to a separate commit, it's not related to `err` fixes.